src/main/java/org/spongepowered/common/bridge/optimization/OptimizedMapDataBridge.java

```diff
@@ -42,4 +42,8 @@
     void mapOptimizationBridge$updateItemFrameDecoration(EntityItemFrame frame);
 
     void mapOptimizationBridge$removeItemFrame(EntityItemFrame frame);
+
+    int getWorldId();
```

Comment:
This is not the appropriate section to put this bridge method.

It needs to go into org.spongepowered.common.bridge.item.MapItemBridge (make the interface if it doesn't exist)

You need to then implement that with a mixin in org.spongepowered.common.mixin.core.item.MapItemMixin

Lastly, this should be get/setDimensionId

Reply:
i dont think a mixin is actually needed, i just put it there because it had
`private static Field mapOptimizationImpl$dimensionField;`
which you can then use to access the dimension field with reflection so spongeforge's patch can work.
Where should I put the code in this case?